src/Aspire.Hosting.Orleans/OrleansService.cs

```diff
@@ -1,34 +1,55 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using Aspire.Hosting.ApplicationModel;
+
 namespace Aspire.Hosting.Orleans;
 
 /// <summary>
 /// Describes an Orleans service.
 /// </summary>
-/// <param name="builder">The distributed application builder.</param>
-/// <param name="name">The service name.</param>
-public class OrleansService(IDistributedApplicationBuilder builder, string name)
+public class OrleansService
```

Comment:
```suggestion
public sealed class OrleansService
```

Do we expect people to derive from this?